tensorflow/python/ops/array_ops.py

```diff
@@ -100,19 +100,34 @@
 
 
 def rank(input, name=None):
-  """Override for gen_array_ops.rank(), making it work for SparseTensor.
+  """Returns the rank of a tensor.
+
+  This operation returns an integer representing the rank of `input`.
+
+  For example:
+
+  ```prettyprint
```

Comment:
Does "prettyprint" marker work?  I see that many Python source files use "python" after the 3 backticks instead.